src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -655,6 +657,7 @@ def test_webapp_backup_config(self, resource_group):
         ]
         self.cmd('webapp config backup show -g {} --webapp-name {}'.format(resource_group, webapp_name), checks=checks)
 
+    @record_only()
```

Comment:
Why not just fix the test then?  Even as @record_only, it will fail if someone actually tries to re-record it.

Reply:
My bad, I thought I have communicated the context to each individual, but appears not.
In short, the sas token used to enable webapp back-up has to be specifically made by app service team. The one made by CLI (or even storage account's portal UI) won't work because of a known service bug. The tracking issue is https://github.com/Azure/azure-cli/issues/5369. 
I have thought of just getting rid of this test, but figured it still has some values like to catch syntax level of bugs caused sdk breaking changes.
I can put in some reasons/comment here to clarify.

Reply:
Then why not just keep the test live but anticipate/expect a failure?